.github/workflows/update-custom-image.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Rebuild All Tagged Custom Cloud Shell Images"
+on:
+  schedule:
+  # run daily at midnight
+    - cron: '0 0 * * *'
+  push:
+    # run on pushes to test branch
+    branches:
+      - build-trigger
+  workflow_dispatch:
+  # trigger through UI or API
+  
+jobs:
+  build-trigger:
+    runs-on: [self-hosted, push-privilege]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run Cloud Build Trigger to Rebuild All Tagged Images
+      run: |
+        gcloud beta builds triggers run --branch=master Build-CloudShell-Image
```

Comment:
Another idea: Maybe after triggering the build, we should do a `gcloud beta builds log` so we can watch the results withing GitHub Actions?

Reply:
Yes, that would be very convenient!